sbc_compassion/models/correspondence_template.py

```diff
@@ -408,24 +407,6 @@ def _compute_template_data(self):
             template.qrcode_y_max = template.page_height * float(
                 config_obj.get_param("qrcode_y_max")
             )
-            if template.pattern_image:
```

Comment:
I think you can also remove the field `pattern_image` from the model and all its usages, because it was only used for template detection if I remember correctly.